geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+# Bond angles:  thetae^se (degrees)
+# Source Paper: Vogt N, Demaison J, Rudolph HD (2014) Accurate equilibrium structures of fluoro- and chloroderivatives of
+methane. Mol Phys 112(22):2873-2883
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C  Cl   1.76425(3)
+C  H    1.0816(2)
+Cl C Cl 112.166(3)
+H  C H  111.772(4)
+H  C Cl 108.237(10)
+
+NAME = CH2FI
+# Point Group: Cs
+# Shape: Seesaw
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+# Source Paper: Puzzarini C, Cazzoli G, López JC, Alonso JL, Baldacci A, Baldan A, Stopkowicz S, Cheng L, Gauss J (2012)
+Rotational spectra of rare isotopic species of fluoroiodomethane: Determination of the equilibrium structure from
+rotational spectroscopy and quantum-chemical calculations. J Chem Phys 137(2):024310/1-024310/11
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C F   1.3610(16)
+C I   2.1396(16)
+C H   1.0831(4)
+F C I 110.50(31)
+H C F 109.74(24)
+H C I 106.98(17)
+H C H 112.84(7)
+
+NAME = CH2F2 2014
+# Point Group: C2v
+# Shape: Seesaw
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+# Source Paper: Vogt N, Demaison J, Rudolph HD (2014) Accurate equilibrium structures of fluoro- and chloroderivatives of
+methane. Mol Phys 112(22):2873-2883
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C F   1.35323(1)
+C H   1.08703(3)
+F C F 108.282(1)
+H C H 113.442(9)
+H C F 108.750(2)
+
+NAME = CH3ClZn 2016
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Min J, Bucchino MP, Kilchenstein KM, Ziurys LM (2016) Rotational spectroscopy of ClZnCH3 (X1
+A1): Gasphase synthesis and characterization of a monomeric Grignard-type reagent. Chem Phys Lett 646(4):174-178
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Zn Cl  2.23(7)
+Zn C   1.9201
+C  H   1.105
+Zn C H 110.2(5)
+H  C H 108.7
+
+# Zn C 1.9201, C H 1.105 and H C H 108.7 fixed to the experimental value of CH3ZnI
+
+NAME = CH3IZn 2014
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Bucchino MP, Young JP, Sheridan PM, Ziurys LM (2014) Structural determination and gas-phase synthesis of
+monomeric, unsolvated IZnCH3 (X1A1): a model organozinc halide. J Phys Chem A 118(47):11204-11210
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Zn I   2.4076(2)
+Zn C   1.9201(2)
+C  H   1.105(9)
+Zn C H 110.2(5)
+H  C H 108.7(5)
+
+NAME = CH3NOS
+# Point Group: C3v
+# Shape: Linear --- Trigonal pyramidal
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Liu X, Xu Y (2011) Infrared and microwave spectra of the acetylene-ammonia and carbonyl sulfide-ammonia
+complexes: a comparative study of a weak C–H⋅⋅⋅N hydrogen bond and an S⋅⋅⋅N bond. Phys Chem Chem Phys
+13(31):14235-14242
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+S...N   3.3233(2)
+NH3 SCO 4.3607(2)
+α       25.23(4)
+
```

Comment:
I understand that "alpha" is all they gave in the book (along with a footnote), but it's not clear to me what it its, and it would be even less clear to the users of this database who haven't seen the footnote. I don't know what the best solution is here, because the footnote isn't enough for me to know immediately what angles this is.